README.md

```diff
@@ -167,6 +167,7 @@ The runtime environment sets two variables by default
 
 ### Other notes
 
+* Around 2022.10.14, the cuda-nvcc was pinned to 11.6.124, to match cudatoolkit version 11.6. This was done to make sure that JAX would work without throwing an error (https://github.com/pangeo-data/pangeo-docker-images/issues/387). 
```

Comment:
Actually, I just double checked and [`cuda-nvcc-11.6.124`](https://github.com/pangeo-data/pangeo-docker-images/blob/770b758a579444f511ab259ed04df7ed77c061ee/ml-notebook/conda-linux-64.lock#L7) is matched with [`cudatoolkit=11.7`](https://github.com/pangeo-data/pangeo-docker-images/blob/770b758a579444f511ab259ed04df7ed77c061ee/ml-notebook/conda-linux-64.lock#L37) in the lockfile. I'm guessing the `nvidia-smi` output at https://github.com/pangeo-data/pangeo-docker-images/issues/387#issuecomment-1276804420 which indicates `CUDA Version: 11.6` is the cuda(toolkit?) version installed on Kubernetes, and `cudatoolkit=11.7` is the one from the docker image?

To be honest, all of these minor version differences (11.6, 11.7) shouldn't matter according to https://stackoverflow.com/questions/53422407/different-cuda-versions-shown-by-nvcc-and-nvidia-smi, since cuda 11 has gotten better with forward and backward compatibility compared to cuda 10. But this sentence probably needs a bit more rewording still :upside_down_face:

Reply:
@weiji14 - I am very new and confused about the terminology here. Could you take a stab at rewording? 
@ngam - had described the main problem here: https://github.com/pangeo-data/pangeo-docker-images/issues/390#issuecomment-1278996477. We had to downgrade cuda-nvcc so it would match the driver version on the hub. I don't know what sets the highest driver number that is avaiable, as @yuvipanda was the one who sets this somewhere (?)?

Reply:
Can you first report the cuda **driver** version on the M2Lines Hub using `nvidia-smi`? I see at https://github.com/pangeo-data/pangeo-docker-images/issues/387#issuecomment-1276804420 that @yuvipanda reported it as `Driver Version: 510.47.03`, but want to confirm if that is still the case. This driver (the connection between the NVIDIA GPU hardware and all the CUDA software libraries) basically dictates what versions of stuff can be used.

Reply:
Ok, will do. 
Is this not set by ?: 
[cuda-nvcc==11.6.124
cudatoolkit==11.7.0](https://github.com/pangeo-data/pangeo-docker-images/blob/118d49782bac9486d07bcc1b6231067473bcfa7b/ml-notebook/packages.txt#L96-L97)

Reply:
At the moment it reads: 
![Screen Shot 2022-10-18 at 11 22 52 AM](https://user-images.githubusercontent.com/18236610/196473518-b98ebaa7-ffd0-4d12-9a26-82cedddc3a19.png)

Reply:
Thanks, if you have access to the K80 GPUs, can you also paste the output of `nvidia-smi` from there?

Reply:
@ngam - I am leaning on the same side as you about this now. 
If this is only a problem for Jax, then maybe we can have the default be an unpinned version. Then leave a note that that if someone wants to solve the jax issue then they can mamba install the version number after checking driver number. What do you all think?